### ISSUE TITLE
Programmatically setting the value of a required select box

### ISSUE THREAD

**Issue by bigerock** [Knockout-Contrib/Knockout-Validation#406]
Programmatically setting the value of a required select box is throwing validation error. 

I've got a select box that's required. The user fills out the form, and I reset it, but then change the value of one of the select boxes. Knockout validation is throwing the 'required' error and thus displaying the error. However, it is properly selected. 

I've tried a few things to disable this behavior:
- set myobservable.isModified(false) - didn't work
- set myobservable(myobservable()) - meaning i've tried setting the value again, didn't work

any ideas?

Basically i want to tell that select box that, no, you're fine, you're set to the correct value and therefore shouldn't show the 'required' error.
